Facebook.Android/component/component.yaml

```diff
@@ -15,8 +15,8 @@ libraries:
     - ../output/Xamarin.Facebook.AudienceNetwork.dll
 packages:
   android:
```

Comment:
Probably easier to just list the 2 Xamarin.Facebook.* NuGets here and let NuGet dependency do its job.